testing/kuttl/e2e/password-change/02-errors.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: password-change-pguser-password-change
+data:
+  verifier: ""
```

Comment:
⛏️ I'd appreciate a comment here about what's being asserted. If I understand correctly, the step passes/finishes when PGO (eventually) fills in the `verifier` field.